kbase-extension/static/kbase/js/common/jobManager.js

```diff
@@ -700,6 +699,7 @@ define(['common/dialogMessages', 'common/jobs', 'common/jobCommChannel'], (
                     batchJob = this.model.getItem('exec.jobState');
                 if (!allJobs || !Object.keys(allJobs).length) {
                     this.model.deleteItem('exec');
+                    this.resetCell();
```

Comment:
add in a 'resetCell' command that alerts the jobCommChannel that the cell with ID `cellId` has finished or cancelled any jobs currently associated with that cell ID